pkg/semconv/datasource.go

```diff
@@ -0,0 +1,42 @@
+package semconv
+
+import "go.opentelemetry.io/otel/attribute"
+
+const (
+	// The UID of the datasource.
+	//
+	// Type: string
+	// Required: No
+	// Stability: stable
+	// Examples: "abcdef-12345"
+	GrafanaDatasourceUIDKey = attribute.Key("grafana.datasource.uid")
```

Comment:
Question: what is the reason to make those public if they are wrapped in the function below?

Reply:
it's what OTel's `semconv` package does, and i am using the same codegen template